tensorflow/compiler/xla/service/dynamic_padder_test.cc

```diff
@@ -15,6 +15,7 @@ limitations under the License.
 
 #include "tensorflow/compiler/xla/service/dynamic_padder.h"
 
+#include "absl/strings/str_replace.h"
```

Comment:
If required, this needs to be a separate PR.

Reply:
compile error without the header

```
INFO: Deleting stale sandbox base /root/.cache/bazel/_bazel_root/efb88f6336d9c4a18216fb94287b8d97/sandbox
ERROR: /root/tensorflow/tensorflow/compiler/xla/service/BUILD:2311:1: C++ compilation of rule '//tensorflow/compiler/xla/service:dynamic_padder_test_gpu' failed (Exit 1)
tensorflow/compiler/xla/service/dynamic_padder_test.cc: In member function 'virtual void xla::{anonymous}::ExecutionTest_ScatterUpdate_Test::TestBody()':
tensorflow/compiler/xla/service/dynamic_padder_test.cc:266:7: error: 'StrReplaceAll' is not a member of 'absl'
       absl::StrReplaceAll(hlo_text, {{"INDICES_BOUND", "2"}});
       ^
tensorflow/compiler/xla/service/dynamic_padder_test.cc:281:7: error: 'StrReplaceAll' is not a member of 'absl'
       absl::StrReplaceAll(hlo_text, {{"INDICES_BOUND", "4"}});
...
```